packages/jellyfish-api-core/__tests__/category/loan/closeVault.test.ts

```diff
@@ -20,6 +20,7 @@ describe('Loan', () => {
   let oracleId: string
 
   beforeAll(async () => {
+    jest.setTimeout(580000)
```

Comment:
hmmm... i think we have issue on `tGroup.get(2)` which takes time to generate utxos

Reply:
you can stick to using 2 nodes which is sufficient
add `await tGroup.get(0).container.waitForWalletBalanceGTE(100000)` after the line `waitForWalletCoinbaseMaturity`

Reply:
arr i guess this is what meant by 'the first 4 groups will be funded xx amount'? if so it all makes sense now

Reply:
wait. ignore. i still dont have how i dont get the prefunded millions dfi